security/pkg/nodeagent/test/README.md

```diff
@@ -0,0 +1,27 @@
+# SDS Local Integration Test
```

Comment:
Looks great :)

Reply:
Is this the good place to put the integration test? I thought these tests are running as unit tests?

Reply:
These tests are kicked by the unit test job, integration test job only start Istio integration test in tests/integration.